cmd/zoekt-archive-index/main.go

```diff
@@ -210,8 +210,10 @@ func main() {
 	flag.Parse()
 
 	// Tune GOMAXPROCS to match Linux container CPU quota.
-	_, _ = maxprocs.Set()
-
+	_, err := maxprocs.Set()
+	if err != nil {
+		log.Printf("could not set GOMAXPROCS: %v\n", err)
+	}
```

Comment:
can you get rid of this change? It results in log spam. We don't only run in containers and linux (eg macbooks)

Reply:
Reverted! This was due to a (likely unfounded) concern about cgroups v2 with newer version of `containerd` 
See https://github.com/uber-go/automaxprocs/pull/44